smapply/api.py

```diff
@@ -232,8 +232,17 @@ def parse_webhook_user(webhook):
     else:
         user = profile.user
     if webhook.award_id is not None:
-        klass_info = SMApplyAPI().get('/awards/{}'.format(webhook.award_id)).json()
-        personal_price = klass_info['price']
+        application_meta = SMApplyAPI().get('/submissions/{}/metadata/{}/'.format(
+            webhook.submission_id,
```

Comment:
should this be `submission_id` or `award_id` ?

Reply:
The personal price is set on the specific application, so it is the submission_id